patterns/2-structured/project-setup/templates/COMMUNICATION-template.md

```diff
@@ -0,0 +1,73 @@
+# COMMUNICATION.md
+
+***
+Put an individual COMMUNICATION.md file tailored to the project in each repo. If project repo ownership is transferred to a different team in the future, they need to be able to access and edit documentation related to the project. This includes documentation that outlines the communication processes that users should use to contact the team.
+
+**Delete top paragraph when this section is filled out.**
+
+## Team Communication
+
+Team Slack channel:
+
+Special Slack Channels: (topic specific and accessible to anyone who would be an external contributor)
+
+Team Email:
+
+## How to Contact Us
+
+The following types of actions can be moved to the appropriate section and more added.
+
+| Action                         | (General team contact)   |
+|--------------------------------|--------------------------|
+| Contact Method                 | (email or slack channel) |
+| Bug report                     |                          |
+| Feature request                |                          |
+| Contribution process questions |                          |
+| MRs after submission           |                          |
```

Comment:
```suggestion
| Merge Requests after submission           |                          |
```

@kschueths I am going ahead with the assumption that MRs meant "Merge Requests".